src/transformers/file_utils.py

```diff
@@ -1229,12 +1224,16 @@ def hf_bucket_url(
         filename = f"{subfolder}/{filename}"
 
     if mirror:
-        endpoint = PRESET_MIRROR_DICT.get(mirror, mirror)
+        logger.warning(
+            "Deprecated: The mirroring is deprecated."
+            "Tuna and BFSU mirrors are no longer available."
+            "We will soon provide a self-hosted alternative."
+        )
         legacy_format = "/" not in model_id
         if legacy_format:
-            return f"{endpoint}/{model_id}-{filename}"
+            return f"{mirror}/{model_id}-{filename}"
```

Comment:
It looks like the `mirror` argument is now used in a different way, which is a breaking change (passing `mirror="tuna"` used to work and doesn't work anymore for instance). We don't do breaking changes before a major release, so this will need to be reverted.